doc/JSON_INFO.md

```diff
@@ -777,6 +777,8 @@ There are six -resist parameters: acid, bash, chip, cut, elec, and fire. These a
 }
 ```
 
+Note that the above example gives floats, not integers, for the vitamins values.  This is likely incorrect; they should be replaced with integers.
```

Comment:
It appears for *materials* that vitamins are floating-point numbers, per `material.cpp`:

https://github.com/CleverRaven/Cataclysm-DDA/blob/a5b97a80871594cdfcf1283f94529eddeb39a694/src/material.cpp#L83-L85

and then later scaled based on the `healthy` value of the item, in `item_factory.cpp` (the `m->vitamin( v.first )` call being the part that gets the floating point value for the material):

https://github.com/CleverRaven/Cataclysm-DDA/blob/a5b97a80871594cdfcf1283f94529eddeb39a694/src/item_factory.cpp#L440-L448

There are several materials using floating-point vitamin values in `data/json/materials.json`. It's not clear to me what all this math works out to without testing in-game materials, but I hope this helps.

Reply:
You may be able to learn something from `tests/stomach_contents_test.cpp` as well. This comment in `eat_all_nutrients` is relevant to your question about RDA:

```cpp
// Vitamin target: 100% DV -- or 96 vitamin "units" since all vitamins currently decay every 15m.
```

Reply:
I don't know if it is bad form to comment on a merged PR that I'm not actively seeking to improve but...

I never modified this PR to reflect wapcaplet's input.  Happy with it being merged, but I wanted to flag this in some way.